src/sardana/tango/pool/CTExpChannel.py

```diff
@@ -215,6 +215,20 @@ def is_Value_allowed(self, req_type):
             return False
         return True
 
+    def read_Timer(self, attr):
```

Comment:
What about adding a `PoolTimerableDevice` in `sardana.tango.pool.PoolDevice`.
It would iherit from `PoolExpChannelDevice` and would be a base class for `CTExpChannel`, `OneDExpChannel` 
 and `TwoDExpChannel`. This way we eliminate the code duplication.